harness/determined/tensorboard/s3.py

```diff
@@ -40,3 +46,13 @@ def sync(self) -> None:
 
             self.client.upload_file(str(path), self.bucket, key_name)
             self._synced_event_sizes[path] = path.stat().st_size
+
+    def delete(self) -> None:
+        self.resource.Bucket(self.bucket).objects.filter(Prefix=str(self.sync_path)).delete()
+
+        # If we're attached to using the low level APIs:
```

Comment:
preference @rb-determined-ai ?

Reply:
The StorageManager code is some of the oldest code we have.  I don't think we have any attachment to the low-level APIs, that's just how it's always been.

Reply:
(so I prefer the easy way)

Reply:
I wasn't sure if it was some weird minio compat thing I didn't know about, but minio looked like it should work with both APIs from what I saw (I'm unsure if it's actually a different API or it's just a nice client side wrapper around the low levels). I'll just use the nice one.

Reply:
nah, our s3 code was written way before we ran it against minio